docs/provider/conjur.md

```diff
@@ -21,11 +21,18 @@ If you set up your Conjur server with a self-signed certificate, we recommend th
 {% include 'conjur-ca-bundle.yaml' %}
 ```
 
-### External secret store with apiKey authentication
+### External secret store
+
+The Conjur provider is configured as an external secret store in ESO. The external secret store is a Kubernetes custom resource that defines the connection to the Conjur server. There are two methods to authenticate to Conjur that are supported by the Conjur provider:
+
+* `apikey` - Uses a Conjur `hostid` and `apikey` to authenticate with Conjur.
+* `jwt` - Uses a JWT token to authenticate with Conjur.
+
+#### Option 1: External secret store with apiKey authentication
```

Comment:
I'd set this as a level-three heading (`###`), and move the heading levels below up one

Reply:
My idea here is that the outline should look like this:
![image](https://github.com/external-secrets/external-secrets/assets/546965/06db2694-104c-40a1-bccb-c5c9bde6e083)
This way External secret store is an L3 heading, with the two options (apikey and jwt) as L4 headings.

Reply:
@szh, yep, makes sense. but adding a comment to the intro paragraph above